certbot/display/ops.py

```diff
@@ -289,3 +289,49 @@ def _gen_https_names(domains):
             domains[-1])
 
     return ""
+
+
+def _get_validated(method, validator, message, default=None, **kwargs):
+    if default is not None:
+        try:
+            validator(default)
+        except errors.PluginError as error:
+            raise AssertionError("Invalid default {0}".format(default))
+
+    while True:
+        code, raw = method(message, default=default, **kwargs)
+        if code == display_util.OK:
+            try:
+                validator(raw)
+                return code, raw
+            except errors.PluginError as error:
+                z_util(interfaces.IDisplay).notification(str(error), pause=False)
+        else:
+            return code, raw
+
+
+def validated_input(validator, *args, **kwargs):
+    """Like ~certbot.interfaces.IDisplay.input, but with validation.
+    
+    :param validator: 
```

Comment:
This should probably be updated to match the `validator` description below or at the very least the docstring of this function should refer to the other function.